openvdb_ax/openvdb_ax/ax.cc

```diff
@@ -0,0 +1,94 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+#include "ax.h"
+#include "ast/AST.h"
+#include "compiler/Compiler.h"
+#include "compiler/PointExecutable.h"
+#include "compiler/VolumeExecutable.h"
+
+namespace openvdb {
+OPENVDB_USE_VERSION_NAMESPACE
+namespace OPENVDB_VERSION_NAME {
+namespace ax {
+
+/// @note Implementation for initialize, isInitialized and unitialized
+///       reamins in compiler/Compiler.cc
+
+void run(const char* ax, openvdb::GridBase& grid)
```

Comment:
I find the wording of the API a little confusing in places - the mix of executable, execute, executor, run, etc.

I also find repeating words harder to mentally parse:

```
VolumeExecutable executable;
executable.execute();
```

Wondering it might read better to "execute" a "binary" rather than executing an executable (ie VolumeBinary instead of VolumeExecutable). 

I find most programming languages go with eval or evaluate instead of run for evaluating an expression too.